pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go

```diff
@@ -43,6 +45,9 @@ const (
 	serviceCheckName        = "snmp.can_check"
 	deviceReachableMetric   = "snmp.device.reachable"
 	deviceUnreachableMetric = "snmp.device.unreachable"
+	pingCanConnectMetric    = "ping.canConnect" //TODO:(ken) What is the right name/prefix here?
+	pingPacketLoss          = "ping.packetLoss"
+	pingAvgRttMetric        = "ndm.avgRtt"
```

Comment:
Since we are running ping within SNMP integration, I'm wondering if it could still make sense to have `snmp` as prefix (even though it's not snmp related call/implementation)

Reply:
The only thing I worry about then is that if we ever split this out to its own thing, then we'll have to continue sending those metrics with the SNMP prefix as well

Reply:
What about this?

```
snmp.ping.can_connect
snmp.ping.packet_loss
snmp.ping.avg_rtt
```